crates/db_schema/src/lib.rs

```diff
@@ -77,6 +78,18 @@ pub enum CommentSortType {
   Top,
   New,
   Old,
+  Controversial,
+}
+
+#[derive(EnumString, Display, Debug, Serialize, Deserialize, Clone, Copy)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// The person sort types. See here for descriptions: https://join-lemmy.org/docs/en/users/03-votes-and-ranking.html
+pub enum PersonSortType {
+  New,
+  Old,
+  MostComments,
+  CommentScore,
```

Comment:
I see there is also a `post_score` and `post_count`. 

You might as well add those now too (since there's a dedicated PersonSortType), as well as indexes for them, just like comment_score has.

![Screenshot_2023-07-14-10-15-54-293_com termux](https://github.com/LemmyNet/lemmy/assets/930215/2b56c689-f7cf-4232-9893-0e1734fc266d)